plugins/@grouparoo/pipedrive/src/lib/common/listMethods.ts

```diff
@@ -86,7 +70,29 @@ async function ensureFieldAndFilter(
   return data.key;
 }
 
-export const GROUP_FIELD_PREFIX = "In Group:";
-function makeGroupFieldName(groupName: string) {
-  return `${GROUP_FIELD_PREFIX} ${groupName}`;
+export async function getGroupFieldKey(
+  client: PipedriveClient,
+  type: EntityType,
+  cacheData: PipedriveCacheData,
+  groupName: string,
+  createIfNotExists?: boolean
+): Promise<string> {
+  groupName = groupName.trim();
+
+  const { appId, appOptions } = cacheData;
+  const cacheDurationMs = 1000 * 60 * 10; // 10 minutes
+  const cacheKey = ["getGroupFieldKey", groupName, appOptions];
```

Comment:
this key needs the `type` so it's different for each.
`["getGroupFieldKey", type, groupName, appOptions];`

Reply:
same for others if I missed them